readme.md

```diff
@@ -229,6 +229,12 @@ yarn closure
 yarn compile-devtools
 ```
 
+## Lighthouse Integrations
+
+* **[Calibre](https://calibreapp.com)** Calibre is a web performance monitoring tool running Lighthouse continuously or on-demand via an API. Test using emulated devices and connection speeds from a number of geographical locations. Set budgets and improve performance with actionable guidelines.
```

Comment:
need to settle on a hyphenation scheme, and maybe bold the related project names too?